test/unit/accord/utils/RandomSource.java

```diff
@@ -301,7 +301,7 @@ default long reset()
 
     default Random asJdkRandom()
     {
-        return new Random()
+        return new Random(nextLong())
```

Comment:
made this change in accord branch awhile ago, so bringing it forward as this method is touched in `sai` tests